### DISCUSSION TITLE
Analysis of the families where not all members were sequenced

### DISCUSSION THREAD

**Discussion by zihhuafang** [broadinstitute/seqr#2252] [Q&A] [answered]
Hi,

Is it possible to use seqr to analyze the pedigrees where not all the family members were sequenced.
For example, only affected offsprings were sequenced but not the parents.

I tried it but got the error message as "The following samples are included in the pedigree file but are missing from the VCF: ...."

Thanks.
Zih-Hua

**Comment by hanars** [comment, marked as answer]
Hi Zih-Hua,

You need to exclude those individuals from the pedigree file - at this time seqr does not support uploading metadata about individuals that are not going to have sequencing data. seqr allows you to edit the pedigree images shown for a family either by editing the image in-app or uploading your own image, so you can still visualize all the individuals in a family even if you don't explicitly load data for them

**Comment by zihhuafang** [comment]
Hi @hanars,

Thanks a lot for the answer. In the case of only affected offsprings sequenced, I should leave the parent ID blank for the siblings then?

**Reply by hanars** [reply]
yes, just make sure the siblings have the same family id

**Reply by zihhuafang** [reply]
Thanks @hanars
